core/renderers/common/info.ts

```diff
@@ -748,4 +748,24 @@ export class RenderInfo {
     this.startY = this.topRow.capline;
     this.bottomRow.baseline = yCursor - this.bottomRow.descenderHeight;
   }
+
+  /** Returns the connection measurable associated with the given connection. */
+  getMeasureableForConnection(conn: RenderedConnection): Connection | null {
+    if (
+      this.outputConnection &&
+      this.outputConnection.connectionModel === conn
```

Comment:
Assuming that `conn` can never be `undefined` (which I think it can't since it's a `RenderedConnection`):
```suggestion
      this.outputConnection?.connectionModel === conn
```